src/Particle/ParticleContainer.H

```diff
@@ -192,6 +192,24 @@ void make_ParticleContainer_and_Iterators (py::module &m, std::string allocstr)
         .def_property_readonly_static("NArrayReal", [](const py::object&){return ParticleContainerType::NArrayReal; })
         .def_property_readonly_static("NArrayInt", [](const py::object&){return ParticleContainerType::NArrayInt; })
 
+        .def_property_readonly("num_runtime_real_comps", &ParticleContainerType::NumRuntimeRealComps)
+        .def_property_readonly("num_runtime_int_comps", &ParticleContainerType::NumRuntimeIntComps)
+        .def_property_readonly("num_real_comps", &ParticleContainerType::NumRealComps)
```

Comment:
So once this is merged we can clean up the `WarpXparticleContainer.cpp` file, for example the line at https://github.com/ECP-WarpX/WarpX/blob/b3ba07df16de4e9736b01a8cc7024740714dcad6/Source/Python/Particles/WarpXParticleContainer.cpp#L88, right?

Reply:
Yes, def!